### PR TITLE
Ps setup express

### PR DESCRIPTION
**Pull request by ps3611**
a working backend,
some parts are not DRY and bit of code duplication with mth-api...
will be improved in the next version

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const router = require('./router');
+const app = express();
+const port = 3001;
+
+app
+  .use(router)
+  .listen(port, () => console.log('Express app listinging on port', port));
```
Comment (alvaroha1) [inline index.js]: Not sure if it's a problem but wasnt ${Port}?

File: router.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const router = require('express').Router();
+const homeController = require('./controllers/homeController');
+
```
Comment (alvaroha1) [inline router.js]: don't we need CORS?

File: models/db.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+
+module.exports = mongoose.createConnection(
+  'mongodb://localhost:27017/mth',
```
Comment (arol) [inline models/db.js]: Better to use environment vars for this.

File: router.js (hunk 2 above)
Comment (arol) [reply router.js]: Yeah but this is an app concern, not in the router.

**Pull request merged**